### PR TITLE
[bitnami/airflow] Replace airflow-web deployment with statefulset to be able to use more than 1 web replica.

### LINKED ISSUE #7178: CSRF Session token is missing in Airflow web UI
**Which chart**:
Airflow / Chart version: 10.2.8

**Describe the bug**
CSRF Session token is missing

csrf

Below is my webserver_config.py file 

```
# Webserver ConfigMap
- apiVersion: v1
  kind: ConfigMap
  metadata:
    name: webserver-configmap
  data:
   webserver_config.py: |-
     import os

     #from flask_appbuilder.security.manager import AUTH_DB
     basedir = os.path.abspath(os.path.dirname(__file__))

     # Flask-WTF flag for CSRF
     WTF_CSRF_ENABLED = True
     #AUTH_TYPE = AUTH_DB
     AUTH_ROLE_PUBLIC = 'Admin'
```
I have also set `WTF_CSRF_ENABLED = True` , not sure if this really helps. 

**To Reproduce**
Run the airflow Helm chart with web server replicas set to more than 2 or 3.

**Expected behavior**
Web server should be visible even web server replicas set to 2 or 3

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
v3.6.2
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2+k3s1", GitCommit:"5a67e8dc473f8945e8e181f6f0b0dbbc387f6fca", GitTreeState:"clean", BuildDate:"2021-06-21T20:52:44Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2+k3s1", GitCommit:"5a67e8dc473f8945e8e181f6f0b0dbbc387f6fca", GitTreeState:"clean", BuildDate:"2021-06-21T20:52:44Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

### PR DESCRIPTION
**Pull request by randradas**
**Description of the change**

Replace airflow-web deployment with a statefulset because of pod readiness order need to be satisfied sequentially when more than one airflow-web pod is required. `podManagementPolicy = OrderedReady` is required.

The reason because you need an ordered web replica escalation if that otherwise several db init can be executed in parallel which leads to errors. db init is a multiple transaction operation and there is not a lock that avoid the race condition when multiple web replicas are running db init in parallel.

**Benefits**

Multiple web replicas can be created.

**Possible drawbacks**

Web replicas creation takes longer because of the `podManagementPolicy` default value (_orderedready_). 

**Applicable issues**

  - fixes #7178

**Additional information**

Currently I'm testing it in GKE before be ready to merge.

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by randradas** [comment]
Answered!

File: bitnami/airflow/Chart.yaml [hunk 1]
```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 11.1.3
+version: 11.2.3
```
[the remark this answers is not in the document]
Comment (randradas) [reply bitnami/airflow/Chart.yaml]: done

**Comment by miguelaeh** [comment]
I @randradas ,
please ping me once this is ready for review, also please check the conflicts

**Comment by randradas** [comment]
I need more time, didn't forget.

**Comment by juan131** [comment]
IMHO this is NOT the right approach to use to fix this issue.

A statefulset is the right solution to be used under the following scenarios:

1. The asset requires a stable network identifier (typical in peer-to-peer setups where every peer receive the list of peers to connect to)
2. The asset requires a unique PV per replica (typical in cluster setups where each node saves different data)
3. The asset requires extra-control on the replicas' creation order ("Pod Management Policies").

The problem to solve here is a race condition that appears when multiple Airflow web replicas try to populate the database simultaneously. We could be tempted to use a statefulset to control the creation order and, therefore, avoid the race condition but that won't prevent the race conditions if Airflow web are reallocated.

This is not the 1st time we face this challenge, we already found similar issues in other web apps in the past (e.g. WordPress). The solution we implemented in the past (and I propose for Airflow too) consists on sharing a RWX volume between every replica and include a "semaphore" that allows the container's logic to detect the database is already populated.

**Pull request closed without merging**